docs/usage.md

```diff
@@ -178,7 +177,7 @@ could be used. The example just logs the resulting svg to the javascript console
         };
 
         var graphDefinition = 'graph TB\na-->b';
-        var graph = mermaidAPI.render('graphDiv', graphDefinition, insertSvg);
+        var graph = mermaid.render('graphDiv', graphDefinition, insertSvg);
     });
```

Comment:
The docs mention that `mermaid` is deprecated. This section is about using `mermaidAPI` instead of `mermaid`.

Reply:
Thanks @chris579. I believe I have implemented your requested changes.

Reply:
I don’t think you should use the bootstrapped `mermaidAPI` inside of `mermaid` but instead use it directly.

Reply:
This section is unclear in the docs. What you have updated is better and more correct than the previous text. I think it can be accepted.

The idea with mermaidAPI and mermaid are separation of the actual bootstrapping part of mermaid with the diagram rendering. With bootstrapping I mean the code in mermaid for checking if the document is loaded, finding the elements to render etc. The rendering code is in mermaidApi, the default bootstrapping that looks for elements with the classname mermaid might not be suitable in all integrations.

If an integrator wants to do their own bootstrapping as in some other mechanism for rendering diagrams he or she is free to do so via the API. This is currently not obvious from the docs.

Reply:
Thank you @knsv for the explanation and @chris579 for reopening the PR. I closed the PR because I was not sure how to proceed.

BTW I love mermaid. This library helped me create flowcharts for [WebdriverIO](https://webdriver.io/docs/flowchartsoverview.html), an open source node.js automation tool.  I had originally created a pdf flowchart to help explain this library but the WebdriverIO core team felt pdf's would be too [difficult to maintain](https://github.com/webdriverio/webdriverio/pull/4579#issuecomment-538762233). Instead they wanted a text based flowcharts which mermaid is able to provide.  

Thanks again for creating such an awesome technology! I will look at the backlog and try to help if I can. :)